easybuild/easyconfigs/s/Sambamba/Sambamba-0.7.1.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
```

Comment:
@lexming I know you've made a lot of changes here, but please retain the earlier author information. Feel free to indicate that you are responsible for recent updates.

Reply:
For the record, this is not  derivative work of `Sambamba-0.6.6.eb`. I only took its description for consistency. I do not mind adding multiple authors as a record of contributors though.

Reply:
@lexming Understood. I am usually guided by whatever `--review-pr` tells me, which is rather a blunt instrument.